static/templates/settings/display_settings.hbs

```diff
@@ -1,39 +1,39 @@
-<div id="display-settings" class="settings-section" data-name="display-settings">
+<div id="{{prefix}}-display-settings" class="settings-section" data-name="display-settings">
     <form class="display-settings-form">
-        <div id="user-lang-settings">
+        <div class="user-lang-settings">
             <!-- this is inline block so that the alert notification can sit beside
             it. If there's not an alert, don't make it inline-block. -->
             <h3 class="inline-block">{{t "Language settings" }}</h3>
-            <div class="alert-notification" id="language-settings-status"></div>
+            <div class="alert-notification language-settings-status"></div>
 
             <div class="input-group">
                 <label class="inline-block title">{{t "Default language" }}</label>
-                <button id="default_language" type="button" class="button btn-link rounded small inline-block">
-                    <span id="default_language_name">{{default_language_name}}</span>
+                <button type="button" class="default_language button btn-link rounded small inline-block">
+                    <span class="default_language_name">{{default_language_name}}</span>
                     <i class="fa fa-pencil"></i>
                 </button>
             </div>
 
             {{> ../default_language_modal language_list=language_list_dbl_col}}
         </div>
 
-        <div id="user-display-settings">
+        <div class="display-settings">
             <h3 class="inline-block">{{t "Display settings" }}</h3>
-            <div class="alert-notification" id="display-settings-status"></div>
+            <div class="alert-notification display-settings-status"></div>
 
             <div class="input-group">
                 <label for="default_view" class="dropdown-title">{{t "Default view" }}
                     {{> ../help_link_widget link="/help/change-default-view" }}
                 </label>
-                <select name="default_view" id="default_view">
+                <select name="default_view" class="default_view">
```

Comment:
Do we want to extend these names to be something more like `setting_default_view` to namespace them a bit better, while we're changing this?

Reply:
Yes we can do this. Would be better in a separate commit as we would need to change other settings using `settings_checkbox`.
Do we want realm settings elements to be renamed also, like `realm_invite_required`?